src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -90,6 +95,17 @@
 				.description("Called when clicking on inventory slot.")
 				.examples("")
 				.since("2.2-Fixes-V10");
+		Skript.registerEvent("Item Despawn", EvtItem.class, ItemDespawnEvent.class, "item[ ][stack] %itemtypes% (despawn[ing]|remove|delete)", "item[ ][stack] (despawn|remove|delete) [[of] %itemtypes%]")
```

Comment:
A better syntax for this would be:
```
(item|%-itemtypes%) (despawn[ing]|remov(e|ing)|delet(e|ing))
item (despawn[ing]|remov(e|ing)|delet(e|ing)) of %-itemtypes%
```